### ISSUE TITLE
Use old Windows App

### ISSUE THREAD

**Issue by cabdaman** [C0untFloyd/roop-unleashed#134]
Is there a way to use the older style windows-based app (i.e. not the web-based one) that was a heavily modified roop?

**Comment by C0untFloyd** [comment]
Certainly there is a way but are you sure you're not missing the sweet, sweet new features? 🥲 
If you know your way around Git the branch is still here, called 'old_deprecated'. You could reuse your current conda env and just checkout the branch. If you're on windows, even easier is to abuse the 2.7.0 installer:
1. Open installer.py with a text-editor
2. Go to Line 35
3. Change the value at the end to `cb106632651a59af17858d6aa6138e6d67033f26` 
4. Save it and run the installer
5. Never update and miss all the goodies and cool stuff the future brings...

**Comment by shoothotshot** [comment]
I already edited line35 in installer.py but when I run window_run.bat it keep launching v2.7.0 in webUI.
v2.0.3 with GFPGAN enhancer work very well for me. still finding the way to revert to the old version.

**Comment by lysxelapsed** [comment]
you need to re-install it with that value changed, not just change the value on the already installed 2.7.0.

**Comment by C0untFloyd** [comment]
You could probably also just download the zipped 2.0.3 release and unzip it over your existing new version -> https://github.com/C0untFloyd/roop-unleashed/archive/refs/tags/v2.0.3.zip

Just face reality, your pc doesn't want you to go back 😄

**Comment by shoothotshot** [comment]
I unzip v2.0.3 files overwrite the new version and this happen when I run it.
https://imgur.com/yrOW9NC
Feel like my pc want me to get over the past and look forward to the future.

**Comment by C0untFloyd** [comment]
Oh yes, the old version downloaded the models from the now abandonded roop repo 😣
Try downloading them manually from https://huggingface.co/CountFloyd/deepfake/tree/main and put them into the models folder.

**Comment by shoothotshot** [comment]
It's work. TY :)

**Comment by shoothotshot** [comment]
Ahh no, It doesn't work. I can launch the app, do preview but it failed when start rendering.
https://imgur.com/a/xZsO4cQ

**Comment by SrGrafoasspee** [comment]
@shoothotshot did you find a way to go back to v2.0.3?

**Comment by shoothotshot** [comment]
@SrGrafoasspee not yet

**Comment by C0untFloyd** [comment]
I just did a QoL Release and fixed the broken model links etc. in the old tkinter version. If you still want to go back, I created a new working installer and attached it to the old release:
https://github.com/C0untFloyd/roop-unleashed/releases/download/v2.0.3/roop_unleashed_v2.0.3_final.zip

Just remember to come back in a few weeks to see the gradio version in with great new features and decent speed.
